### PR TITLE
Voron V0 Magnetic Tophat clips

### PR DESCRIPTION
**Pull request by Coole-Guus**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This mod adds clips for magnetic alignment for the voron v0 tophat.

#### How was it tested? 
physically printed and tested it. discord link to video: https://discord.com/channels/460117602945990666/635687829254701107/995345597949292544

#### Any background context you want to provide?
This mod is for low headroom spaces like putting your v0 in tight spaces where you cant fit a hinged solution. It could also help with hinged solutions to align the other half of the hat to the frame. Also pulls the hat towards the frame to create a better seal.

#### Screenshots (if appropriate)
See discord videos: https://discord.com/channels/460117602945990666/635687829254701107/995345597949292544

### CODE REVIEW

**Review by newphreak — CHANGES_REQUESTED** [review]
Thanks for the mod, this looks like a great addition for v0.1 users.
Can you remove the empty README.md in /images/?
Also not fully sure about the .zip, that's not commonly included, but i'll let others comment on that.
All files look printable.
Would love to see a little more instructions on the install or just 2-3 photos. I assume it uses the bom screws for the tophat, but I shouldn't have to assume.
It's also not compatible with V0.0 so state V0.1 somewhere.

**Comment by Coole-Guus** [comment]
> 

thanks for looking at my pr, i am currently not near my printer and will try to add more detailed instructions + photos of the physical install tomorrow (24-7-2022).

**Review by newphreak — APPROVED** [review]
That looks much better. Thanks.

**Comment by Coole-Guus** [comment]
When will this be merged?

**Comment by newphreak** [comment]
> When will this be merged?
When the maintainers get around to it. They are mostly preoccupied with life currently. So be patient and it will be merged when they have a hot minute to look around here again. 👍

File: printer_mods/Dreg/V0_Magnetic_Tophat/README.md
Comment (cbc02009) [inline printer_mods/Dreg/V0_Magnetic_Tophat/README.md:3]: Typo, `usefull` => `useful`

File: printer_mods/README.md
Comment (cbc02009) [inline printer_mods/README.md:128]: This needs to be above DrGhetto's mods

File: printer_mods/Dreg/V0_Magnetic_Tophat/README.md
Comment (cbc02009) [inline printer_mods/Dreg/V0_Magnetic_Tophat/README.md:4]: Typo, `dont` => `don't`

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by cbc02009** [comment]
Sorry for the delay. Life has been very crazy the last couple months. Just a couple small changes and this should be good to go.

**Comment by Coole-Guus** [comment]
no worries that it had to wait. I also kinda forgot about the pull request lol. i fixed the issues but can't seem/ am not good enough at github to fix the merge conflict. can you help with that ?

**Comment by newphreak** [comment]
> no worries that it had to wait. I also kinda forgot about the pull request lol. i fixed the issues but can't seem/ am not good enough at github to fix the merge conflict. can you help with that ?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
Go to your fork and just doublecheck that it's up to date

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**
